### DISCUSSION TITLE
A question about the telepathic circuit.

### DISCUSSION THREAD

**Discussion by Insomniac314** [eccentricdevotion/TARDIS#926] [Q&A]
So I am inside my friend's TARDIS as a companion, I had him turn on the telepathic circuit so that I can input commands. I shift-right click the telepathic circuit which prompted me to enter a /tardis command. I entered /tardistravel world 0 64 0, but the server tells me that Im not in my own tardis. 

What my goal was to set coordinates via the telepathic circuit so that my friend can go to them. Is this a bug?

**Comment by eccentricdevotion** [comment]
Your friend needs to be travelling with you in their TARDIS.

**Reply by Insomniac314** [reply]
I was inside their tardis

**Reply by eccentricdevotion** [reply]
But were they?

**Reply by Insomniac314** [reply]
Yes

**Reply by Insomniac314** [reply]
Update: I got rid of my own tardis, got into my friend's tardis, shift right clicked the telepathic circuit, and ran /tardistravel world 0 64 0.

I get "You do not own a TARDIS"

**Comment by eccentricdevotion** [comment]
Seems as though the PlayerCommandPreprocessEvent no longer sets the player running the command to the time lord of the TARDIS...
I've changed the code to make it work again.
